frontend/tests/playwright/msw/index.ts

```diff
@@ -15,7 +17,8 @@ export const setupMocks = async () => {
     * Therefore, we only set up MSW for the server, and not the browser.
```

Comment:
I the comment above is out of date now.

Reply:
Ah that latter half should probably get removed in #1289. I updated the issue to add that as a **To Do.**

Reply:
Addressed in [01dfc69](https://github.com/iFixit/react-commerce/pull/1275/commits/01dfc6974c1f134202aaacae4a5aaf5ed0be0f8a). We will be making some changes to this file with #1289 so I updated the comment.